styleguide/styleguide.xml

```diff
@@ -851,6 +875,28 @@ if(theThing != done)
 }
 </CODE_SNIPPET>
 <p>The reasoning here is that because we do not require braces for single-line bodies, the braces should be clearly visible when they are present.</p>
+<p>The only exceptions are macro calls that are only found in the unit tests:</p>
+<CODE_SNIPPET>
+SCENARIO( "Creating an Account" , "[Account][Creation]" ) {
+    GIVEN( "an account" ) {
+        Account account;
+        WHEN( "money is added" ) {
+            REQUIRE( account.Credits() == 0. );
+            account.AddCredits(100);
+            THEN( "the balance is increased" ) {
+                REQUIRE( account.Credits() == 100 );
+            }
+        }
+        WHEN( "a fine is levied" ) {
+            REQUIRE( account.TotalDebt() == 0 );
+            account.AddFine(10000);
+            THEN ( "debt is incurred" ) {
+                REQUIRE( account.TotalDebt() == 10000 );
+            }
+        }
+    }
+}
+</CODE_SNIPPET>
```

Comment:
Can you shorten both code snippets a bit so that it only shows the current rule being discussed?